Find a functionally equivalent code:com.android.keyguard.FaceUnlock.handlePokeWakelock:COMMENT
Method Modifier: 
Comment:/**
 * If the screen is on, pokes the wakelock to keep the screen alive and active for a specific
 * amount of time.
 */

@@ -1,6 +1,6 @@
 {
     PowerManager powerManager = (PowerManager) mContext.getSystemService(Context.POWER_SERVICE);
     if (powerManager.isScreenOn()) {
-        mKeyguardScreenCallback.userActivity(millis);
+        mKeyguardScreenCallback.userActivity();
     }
 }

